omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/PatientIdentifierController1_9Test.java

```diff
@@ -114,6 +115,33 @@ public void shouldEditAPatientIdentifier() throws Exception {
 	}
 	
 	@Test
+	public void shouldUnsetOtherPreferredIdentifiers() throws Exception {
+		PatientIdentifier nonPreferred = service.getPatientIdentifierByUuid(getUuid());
```

Comment:
The nonPreferred variable name is a bit confusing. Can you think of better name?
For instance, look at this on line 121 assertTrue(nonPreferred.isPreferred());
In other words, how can non preferred be expected to be preferred? 😊 
Do you agree?

Reply:
Yes, makes sense. Changed to exisitingIdentifer